js/components/controlBar.js

```diff
@@ -317,6 +317,26 @@ var ControlBar = createReactClass({
     this.props.controller.toggleMoreOptionsScreen(this.moreOptionsItems);
   },
 
+  filterAudioControls: function() {
+    //We receive location param in desktopContent, instead of audioOnly.
+    //This is necessary to display or not the button, depending on Backlot settings.
+    var defaultConfig = this.props.skinConfig.buttons.audioOnly;
+    var configItems = this.props.skinConfig.buttons.desktopContent;
+    for (var k = 0; k < configItems.length; k++) {
+      var configName = configItems[k].name;
+      var disableButton = configItems[k].location === "none";
+      if (disableButton) {
+        for (var j = 0; j < defaultConfig.length; j++) {
+          if (defaultConfig[j].name === configName) {
+            defaultConfig[j].location = "none";
+            break;
+          }
+        }
+      }
+    }
```

Comment:
```suggestion
this.props.skinConfig.buttons.desktopContent.forEach(item => {
  if (item.location !== 'none') {
    return;
  }
  defaultConfig.filter(
    field => field.name !== item.name
  ).forEach(
    field => field.location = 'none'
  );
});
```
ES6 made JS much more descriptive and less talkative. That was a great job of JS community. Let's utilize it.